src/anybuf.rs

```diff
@@ -407,11 +408,12 @@ impl Anybuf {
     ///
     /// // Three bytes fields with field number 5
     /// let serialized = Anybuf::new()
-    ///     .append_repeated_bytes(5, &[b"", b"abcd", &blob])
+    ///     .append_repeated_bytes::<&[u8]>(5, &[b"", b"abcd", &blob])
```

Comment:
If putting generic is annoying, you could avoid it by putting slice as first element:
```suggestion
    ///     .append_repeated_bytes(5, &[blob.as_slice(), b"", b"abcd"])
```

Reply:
Nice, done